### ISSUE TITLE
Reproducing the results in the paper

### ISSUE THREAD

**Issue by GKIBMNY** [lucidrains/CoCa-pytorch#5]
Thanks for this repo. Curious, is this an independent implementation of the CoCa paper? If yes, did you reproduce any result in the paper to ensure correctness of implementation?

**Comment by Camellia-tx** [comment]
Hi, I am also reproducing this model, did you succeed in reproducing it? I'm experiencing the problem that the contrastive loss never changes while training, have you encountered this problem?

**Comment by lucidrains** [comment]
@Camellia-tx they have already reproduced it over at OpenClip https://laion.ai/blog/coca/
